### PR TITLE
Use etstool module to install/test pkg

### PR DESCRIPTION
**Pull request by rahulporuri**

### CODE REVIEW

**Comment by rahulporuri** [comment]
the env installation proceeds as expected but the tests themselves aren't run

```
$ edm run -- python -m etstool test
Running tests in 'ensemble-test-2.7-pyside'
copying file to tempdir: .coveragerc
[EXECUTING] edm run -e ensemble-test-2.7-pyside -- coverage run -p -m nose.core -v ensemble --nologcapture
----------------------------------------------------------------------
Ran 0 tests in 0.000s
OK
```

**Comment by rahulporuri** [comment]
@prabhuramachandran any idea why these errors happen and what the solution might be - 

```

test_data_update (ensemble.volren.tests.test_volume_renderer.VolumeViewerTestCase) ... Generic Warning: In /home/vagrant/workspace/Buildsystem/test/pisi/tmp/VTK-7.0.0-3/work/VTK-7.0.0/Rendering/Volume/vtkVolumeRayCastMapper.cxx, line 118
vtkVolumeRayCastMapper::vtkVolumeRayCastMapper was deprecated for VTK 7.0 and will be removed in a future version.
ERROR: In /home/vagrant/workspace/Buildsystem/test/pisi/tmp/VTK-7.0.0-3/work/VTK-7.0.0/Rendering/OpenGL2/vtkOpenGLRenderWindow.cxx, line 545
vtkXOpenGLRenderWindow (0x4093be0): GL version 2.1 with the gpu_shader4 extension is not supported by your graphics driver but is required for the new OpenGL rendering backend. Please update your OpenGL driver. If you are using Mesa please make sure you have version 10.6.5 or later and make sure your driver in Mesa supports OpenGL 3.2.
ERROR: In /home/vagrant/workspace/Buildsystem/test/pisi/tmp/VTK-7.0.0-3/work/VTK-7.0.0/Rendering/OpenGL2/vtkShaderProgram.cxx, line 369
```

see travis job - https://travis-ci.org/enthought/ensemble/builds/452650389 

Is it because of the vtk eggs that we are building?

Note that we saw similar errors on virtualcore when we used ensemble master + mayavi latest release + vtk 7 (edm egg)

**Comment by prabhuramachandran** [comment]
The OpenGL version on the travis build machines does not support the newer OpenGL2 backend that is the default with VTK 7.x onwards.  I suspect there is a way to get better opengl support on the travis hosts or have a suitable mesa version installed but I haven't done that myself so do not know the answer off-hand.  If you do find out a good way to setup the travis hosts with an better OpenGL version, I would love to know though.

File: etstool.py [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+    "mayavi",
+    "mock",
+    "nose",
+    "pyface",
+    "pyside",
+    "scipy",
+    "traits",
+    "traits_enaml",
+    "traitsui",
+    "vtk"
```
Comment (jwiggins) [inline etstool.py]: Use a trailing comma here, so that future diffs don't need to touch this line.

**Comment by rahulporuri** [comment]
ok. the tests themselves pass but there's a weird traceback, which is being interpreted as an error, causing travis to say the tests failed - 

```

X Error of failed request:  BadDrawable (invalid Pixmap or Window parameter)
  Major opcode of failed request:  14 (X_GetGeometry)
  Resource id in failed request:  0x20002c
  Serial number of failed request:  53
  Current serial number in output stream:  53
Command '['edm', 'run', '-e', 'ensemble-test-2.7-pyside', '--', 'coverage', 'run', '-p', '-m', 'nose.core', '-v', 'ensemble', '--nologcapture']' returned non-zero exit status 1
The command "edm run -- python -m etstool test" exited with 1.
```

**Comment by rahulporuri** [comment]
Note that the error isn't seen when the individual tests are run in `ensemble.volren.tests.test_volume_renderer.VolumeViewerTestCase`. It's only noticed when the full test suite is run.

Also note that error is reported to happen during app exit - http://vtk.1045678.n5.nabble.com/GLXBadDrawable-on-application-exit-td5714081.html

**Pull request merged**